foodhub/accounts/views.py

```diff
@@ -0,0 +1,239 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import generics
+from accounts.models import (Merchant, Item, Store, Order, User)
+from accounts.serializers import (MerchantSerializer, ItemSerializer,
+                                  StoreSerializer, OrderSerializer, RegisterSerializer, LoginSerializer)
+from django.contrib.auth import authenticate
+from rest_framework_simplejwt.views import TokenObtainPairView
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
+from rest_framework_simplejwt.tokens import RefreshToken, AccessToken
+from rest_framework import permissions
+from re import search
+from rest_framework import viewsets, filters, status
+import json
+import os
+from rest_framework.settings import api_settings
+from json.encoder import JSONEncoder
+from rest_framework.pagination import LimitOffsetPagination
+from django.http.response import JsonResponse
+from rest_framework.response import Response
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from accounts.tasks import create_order
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+import structlog
+from rest_framework.utils.urls import remove_query_param, replace_query_param
+# Create your views here.
+
+log = structlog.get_logger()
+
+class RegisterView(APIView):
```

Comment:
why did u preferred class based view instead of function based view.

Reply:
As per requirement i have updated my views.py file